homeassistant/const.py

```diff
@@ -343,6 +343,12 @@
 TEMP_CELSIUS = "°C"
 TEMP_FAHRENHEIT = "°F"
 
+# Time units
+TIME_SECONDS = "s"
+TIME_MINUTES = "min"
+TIME_HOURS = "h"
+TIME_DAYS = "d"
+
```

Comment:
Can we forward think? and add already:

TIME_MONTHS = "m"
TIME_YEARS  = "y"

Reply:
Sure thing!

Reply:
Done https://github.com/home-assistant/home-assistant/pull/32065/commits/661b915c3c667d124c142524695cb34e1fa84737